### ISSUE TITLE
ARIA attributes for `Input Search` component

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#4029]
Scope
---
The `p-input-search` component is often used as a combobox with custom dropdown used for search results. In this scenario some important ARIA attributes are missing, e.g. `aria-expanded` or change the role to `role="combobox"`. 

We should investigate which ARIA attributes are useful and can be added to the semantic `input` elemen inside the shadowDOM.

Examples usage in the wild: 
- https://ask.porsche.com/de/de-DE 
- https://configurator.porsche.com/en-US/mode/model/9921B2/dealer-search?contact-type=form&ab-test-variant=Original

Out of Scope
---
Some ARIA attributes may not be added due to shadowDOM, e.g.: `aria-activedescendant` and `aria-controls`.

Notes  
---
Investigate if other `Input` components are affected too and need the possibility to have ARIA attributes set by the user or if we should add ARIA attributes in general to the basic `input` element to make them available to all kinds of `Input` components.

Acceptance criteria
---
- [x] The inner input element should be adaptable by the user to set ARIA states, roles and properties with an `aria` property

Subtasks
---
- [ ] Task
